packages/web/src/app/shared/component-documentation/ceg/code-generator/code-viewer/code-viewer.component.html

```diff
@@ -1,12 +1,17 @@
 <div class="code-container">
-  @if (!hideTabs) {
-    <div class="top-menu e-theme-dark">
+  <div class="top-menu e-theme-dark">
+    @if (!hideTabs) {
       <elvia-segmented-control
         [size]="(breakpointService.matches(['md', 'lg', 'xl']) | async) ? 'medium' : 'small'"
         [items]="tabs | tabToSegmentedControlItem"
         [value]="activeTabIndex"
         (valueOnChange)="tabIndexChange.emit($any($event).detail.value)"
       />
+    }
+    @if (fileName !== '') {
+      <div class="file-name e-text-small-mono">{{ fileName }}</div>
+    }
```

Comment:
Can this be simplified to just `@if (fileName)` as non-empty strings are truthy?

```suggestion
    @if (fileName) {
      <div class="file-name e-text-small-mono">{{ fileName }}</div>
    }
```